npm_and_yarn/lib/dependabot/npm_and_yarn/helpers.rb

```diff
@@ -555,6 +555,11 @@ def self.package_manager_run_command(name, command, fingerprint: nil)
         result
       rescue StandardError => e
         Dependabot.logger.error("Error running package manager command: #{full_command}, Error: #{e.message}")
+        if e.message.include?("\e[38;5;111mResponse Code\e[39m: \e[38;5;220m404\e[39m (Not Found)\n") &&
+           e.message.include?("The remote server failed to provide the requested resource")
```

Comment:
can this be simplified too just:

```suggestion
        if e.message.include?("The remote server failed to provide the requested resource")
```

if not we could use regex for `"\e[38;5;111mResponse Code\e[39m: \e[38;5;220m404\e[39m (Not Found)\n"` to remove the ansi escape sequences?

Reply:
Thanks and updated as per the review comment.